extensions/dozer/deployment/src/main/java/org/apache/camel/quarkus/component/dozer/deployment/DozerProcessor.java

```diff
@@ -110,6 +110,9 @@ void configureCamelDozer(BuildProducer<ReflectiveClassBuildItem> reflectiveClass
                 Time[].class,
                 Timestamp[].class,
                 URL[].class,
+                Date.class,
+                File.class,
+                URL.class,
```

Comment:
I'm quite beginner with dozer :) Is there really a finished set of classes to be registered as native ? Won't we end up registering every new classes that developers would eventually use in their scenarios ?

Reply:
@aldettinger 
Date is required for https://github.com/DozerMapper/dozer/blob/master/core/src/main/java/com/github/dozermapper/core/converters/DateConverter.java.
The other classes are special. You can see them for example here: https://github.com/DozerMapper/dozer/blob/master/core/src/test/java/com/github/dozermapper/core/functional_tests/builder/PrimitiveTest.java#L138-L140. No other classes should be required. Other classes are dynamically registered by DozerProcessor.

Reply:
I also added a custom class into the test to be sure, that there is no need to register it.

Reply:
ok, I see looks good :+1: